spacemacs/extensions/spacemacs-theme/README.md

```diff
@@ -1,6 +1,6 @@
 # Spacemacs-theme
 
-![spacemacs-theme](/../screenshots/spacemacs-theme-screenshot.png?raw=true)
+![spacemacs-theme](https://github.com/nashamri/spacemacs-theme/blob/screenshots/spacemacs-theme-screenshot.png?raw=true)
```

Comment:
It is better to keep it in @nashamri repo since this extension will be replaced by a MELPA package at some point. I don't want to import useless files in the repo (I should have used a sub-module for this extension :-))